### PR TITLE
1.9.0-RC1 changelog

### PR DESCRIPTION
**Pull request by techknowlogick**
as title

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,324 @@ This changelog goes through all the changes that have been made in each release
...
+* MISC
+  * Update npm dependencies and various tweaks (#7344)
+  * Fix updated drone file (#7336)
+  * Add 'npm' and 'npm-update' make targets and lockfile (#7246)
+  * Add state param to milestone listing API (#7131)
+  * Handle early git version's lack of get-url (#7065)
+  * Fixes #5960 - Adds API Endpoint for Repo Edit (#7006)
+  * Correctly adjust mirror url (#6593)
+  * Drop the bits argument when generating an ed25519 key (#6504)
+  * Content API for Creating, Updating, Deleting Files (#6314)
```
Comment (mrsdizzie) [inline CHANGELOG.md]: This feels like it should be under FEATURE

Comment (techknowlogick) [reply CHANGELOG.md]: lolya, you are right. Seems the automated tool had an issue with these PRs. Some others should also be moved out of this section. Let me make those changes now.

**Review by zeripath — APPROVED** [review]
As per @mrsdizzie otherwise LGTM

**Comment by zeripath** [comment]
Thanks @Techknowlogick!

**Review by lafriks — APPROVED** [review]

**Comment by zeripath** [comment]
Woot!

**Pull request merged**